res/css/photon/message-bar.css

```diff
@@ -5,53 +5,68 @@
 /* See https://design.firefox.com/photon/components/message-bars.html */
 .photon-message-bar {
   display: flex;
-  width: 100%;
   min-height: 32px;
   box-sizing: border-box;
   align-items: center;
-  padding: 0 4px;
+
+  /* Note: 32px is: 4px (left padding for the message bar) + 4px (left padding
+   * for the icon) + 16px (icon width) + 4px (right padding for the icon) + 4px
+   * (space between icon and text) */
+  padding: 4px 4px 4px 32px;
+
+  /* Note: 8px is: 4px (left padding for the message bar) + 4px (left padding
+   * for the icon). And same for the top positioning, because we want the icon
+   * to stick at the top when the text is multiline */
+  background: url(../../img/svg/info-icon.svg) no-repeat 8px 8px / 16px 16px
+    var(--grey-20);
   border-radius: 4px;
+  color: var(--grey-90);
   font-size: 13px;
   font-weight: 400;
   line-height: 1.4;
 }
 
-.photon-message-bar .photon-button {
-  margin-left: 8px;
+.photon-message-bar-action-button {
+  flex: none;
+  margin: 0 8px;
 }
 
-.photon-message-bar .photon-message-bar-close-button {
+.photon-message-bar-close-button {
   width: 24px;
   height: 24px;
+  flex: none;
   margin-left: auto;
   background: url(../../img/svg/close-dark.svg) no-repeat center / 16px 16px;
-  fill: currentColor;
 }
 
-.photon-message-bar-error {
-  /* Note: 32px is: 4px (left padding for the message bar) + 4px (left padding
-   * for the icon) + 16px (icon width) + 4px (right padding for the icon) + 4px
-   * (space between icon and text) */
-  padding: 4px 4px 4px 32px;
+/* This rule is useful only for message bars that have a close button but no
+ * action button. */
+.photon-message-bar-inner-content {
+  margin-right: 4px;
+}
 
-  /* Note: 8px is: 4px (left padding for the message bar) + 4px (left padding
-   * for the icon) */
-  background: url(../../img/svg/error.svg) no-repeat 8px center / 16px 16px
-    var(--red-60);
+.photon-message-bar-error {
+  background-color: var(--red-60);
+  background-image: url(../../img/svg/error.svg);
```

Comment:
This is simplified because most other properties are inherited. This makes it easier to change them in one place only.